mathjax3-ts/output/chtml/Wrappers/mfrac.ts

```diff
@@ -124,27 +123,10 @@ export class CHTMLmfrac<N, T, D> extends CHTMLWrapper<N, T, D> {
 
     };
 
-    protected bevel: CHTMLmo<N, T, D> = null;
+    public bevel: CHTMLmo<N, T, D>;
```

Comment:
Should this have a default value?

Reply:
No, it is initialized in `common/Wrappers/mfrac.ts`.  This is only here to give it the CHTML-specific type so I don't have to coerce it when it gets used.